onie-mk-demo.sh

```diff
@@ -93,6 +94,7 @@ sed -i -e "s/%%DEMO_TYPE%%/$demo_type/g" \
        -e "s/%%ONIE_IMAGE_PART_SIZE%%/$onie_image_part_size/" \
        -e "s/%%EXTRA_CMDLINE_LINUX%%/$EXTRA_CMDLINE_LINUX/" \
        -e "s@%%OUTPUT_RAW_IMAGE%%@$output_raw_image@" \
+       -e "s/%%ONIE_MACHINE%%/$machine/g" \
```

Comment:
>ONIE_MACHINE [](http://example.com/codeflow?start=15&length=12)

Naming is confusing.

`onie_machine` is an old concept in the world of ONIE bootloader, it means what is the running hardware.

What you are trying to name, is the target ASIC vendor related to SONiC image.